checkbox-radio.scss

```diff
@@ -63,8 +63,8 @@
 	@-moz-document url-prefix() {
 		& {
 			appearance: $appearance;
-			height: auto;
-			width: auto;
+			height: 1.1rem;
+			width: 1.1rem;
```

Comment:
I think the input element will actually get a `1.1rem` height already (from the above code), so you may just be able to remove the height/width of `auto`?